docker/php/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM php:7.3-fpm
+
+RUN echo 'Acquire::http::Pipeline-Depth "0";' > /etc/apt/apt.conf.d/00no-pipeline
+ENV TZ=Europe/London
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+RUN apt-get update && \
```

Comment:
Can we install the minimum required to this image please? I think it only needs a few PHP extensions

Reply:
reduced the list by alot